apps/docs/theme.config.tsx

```diff
@@ -90,33 +90,33 @@ const config: DocsThemeConfig = {
         title: 'ClassroomIO Documentation',
         description: 'ClassroomIO is the easiest place to launch and scale your online bootcamp.',
         images: [
+          // {
+          //   url: `${process.env.imagePath}/img/logo-16.png`,
+          //   width: 16,
+          //   height: 16,
+          //   alt: 'ClassroomIO Docs Og Image Alt',
+          //   type: 'image/png'
+          // },
+          // {
+          //   url: `${process.env.imagePath}/img/logo-32.png`,
+          //   width: 32,
+          //   height: 32,
+          //   alt: 'ClassroomIO Docs Og Image Alt',
+          //   type: 'image/png'
+          // },
+          // {
+          //   url: `${process.env.imagePath}/img/new-logo.svg`,
+          //   width: 192,
+          //   height: 192,
+          //   alt: 'ClassroomIO Docs Og Image Alt',
+          //   type: 'image/png'
```

Comment:
please uncomment this. You can't comment  these different resolutions of images, they are used by the browser.

Reply:
yea i am currently reverting all commits to follow up on it based on your reviews